### PR TITLE
[ORCID] Moving slack alert to an included file

### PR DESCRIPTION
**Pull request by carolyncole**
The goal would be to eventually have a singe place to update slack alerts if desired

We are using an old slack token that can only alert to `ansible-alerts`: `vault_pul_slack_token`.  I think we should be moving towards utilizing  `vault_tower_slack_token`

This could just be a search and replace, but it seemed nice to move the slack update to a central location.  I only updated ORCID for now, but all playbooks could be updated to utilize the `include_tasks` once this is merged.

refs #4443 

Code bing run: https://pulibrary.slack.com/archives/C9Z7D6D4G/p1699025104773019

### CODE REVIEW

File: playbooks/slack_playbook_complete.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: tell everyone on slack you ran an ansible playbook
+  community.general.slack:
+    token: "{{ vault_tower_slack_token }}"
+    msg: "Ansible ran `{{ ansible_play_name }}` on {{ inventory_hostname }}"
+    channel: "{{ item }}"
+  loop: "{{ slack_alerts_channel is string | ternary([slack_alerts_channel], slack_alerts_channel) }}"
```
Comment (carolyncole) [inline playbooks/slack_playbook_complete.yml:7]: takes care of the fact that the channel could be a string or array

**Review by acozine — COMMENTED** [review]
+1 for this - it would make future changes to the standard slack message much easier.

The `loop` that handles either a string or an array is unfamiliar to me - I'd love to see it in action.

I find the name of the new file misleading. I'd change it from `playbooks/slack_playbook_complete.yml` to something like `playbooks/slack_tasks.yml`, because with only tasks (no `hosts` entry, etc.) that file would not run as a playbook on its own.

**Comment by carolyncole** [comment]
@acozine I meant it as the slack tasks for the completion of a playbook.  What would make that more clear? `slack_task_for_end_of_playbook` `slack_end_of_run` (something else)?

**Comment by carolyncole** [comment]
@acozine it's the tunery that allows it to either be an array or a single item `ternary([slack_alerts_channel], slack_alerts_channel)`

**Review by VickieKarasic — APPROVED** [review]
Looks good - thanks @kayiwa!

**Pull request merged**